PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx

```diff
@@ -1298,6 +1304,9 @@ void AliAnalysisTaskGammaCalo::UserCreateOutputObjects(){
     fHistoPionSpectrum                      = new TH1F*[fnCuts];
     fHistoProtonSpectrum                    = new TH1F*[fnCuts];
     fHistoKaonSpectrum                      = new TH1F*[fnCuts];
+    fHistoNPionSpectrum                     = new TH1F*[fnCuts];
```

Comment:
could you encapsulate all sphericity histograms also here in the if() condition? The "new" command will already here allocate memory which we would like to avoid in non-sphericity running of the task.

Reply:
Done